### ISSUE TITLE
Add white-space: nowrap to read-more link

### ISSUE THREAD

**Issue by demetris** [justintadlock/exhale#4]
```css
.entry__more-link {
    white-space: nowrap;
}
```

Also, I see that there is no space between the ellipsis entity and the anchor. Is this deliberate?

**Comment by justintadlock** [comment]
Why add this CSS?

> Also, I see that there is no space between the ellipsis entity and the anchor. Is this deliberate?

Different styleguides have different rules on this.  I haven't 100% decided whether I want no spaces, thin spaces, or normal spaces before/after.  Generally, I follow AP style, which I believe (I need to check) uses spaces before/after.

The only thing I'm for sure on is using `&hellip;` rather than three dots, spaced or unspaced.

**Comment by demetris** [comment]
Without the nowrap you get a hanging arrow sometimes.

In general, I don't mind widow words in web typography, but a widow arrow bugs me immensely.

If you want to avoid the widow, it would be perhaps  be better to just attach the arrow to the previous word, but I am not fond of any of the options for doing that: non-breaking space, extra span, or JavaScript.

So I'm suggesting to add nowrap to the element that is already there.

**Comment by justintadlock** [comment]
Not wrapping the entire more link has its own problems, such as having a huge chunk of text missing from the previous line.

A non-breaking space would probably be most appropriate.  I think I'd rather do this since we have direct control over the text.

**Comment by demetris** [comment]
> A non-breaking space would probably be most appropriate.

Yeah. On second thought, the non-breaking space seems fine to me too.

I think I would prefer adding nowrap to the whole link because I see this particular link more like a button.

If you see it as what it really is, a link, then the non-breaking space is more appropriate.
